editor/editor_properties.cpp

```diff
@@ -2754,6 +2770,20 @@ void EditorPropertyResource::set_use_sub_inspector(bool p_enable) {
 	use_sub_inspector = p_enable;
 }
 
+String EditorPropertyResource::_get_file_script_name_or_default(const RES &p_resource) const {
+	Ref<Script> rscript = p_resource->get_script();
+	if (rscript != 0) {
```

Comment:
For consistency/clarity, I would recommend making this be `if (rscript.is_valid())` or something.